AirCasting/Utils/NetworkChecker.swift

```diff
@@ -0,0 +1,37 @@
+// Created by Lunar on 08/07/2021.
+//
+
+import Foundation
+import Network
+
+protocol CheckNetwork {
```

Comment:
"check network" is more of a function name because of it using a verb. We should name our classes using nouns and protocols using nouns or adjectives (like `Presentable` or `Presenter`)